client/mobrender/components/mobilization.js

```diff
@@ -78,7 +75,11 @@ class Mobilization extends React.Component {
                 href='http://www.bonde.org/'
                 className='right my2'
                 target='_blank'>
```

Comment:
@igr-santos It is breaking the mobilization public view rendering. Follow the error:

```
{ SyntaxError: client/components/navigation/sidenav/logo-icon-no-border.svg: Unexpected token (1:1)
> 1 | <?xml version="1.0" encoding="UTF-8" standalone="no"?>
    |  ^
  2 | <!-- Generator: Adobe Illustrator 19.2.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
  3 |
  4 | <svg
    at Parser.pp$5.raise (/Users/gabrielrtakeda/code/bonde-client/node_modules/babylon/lib/index.js:4246:13)
    at Parser.pp.unexpected (/Users/gabrielrtakeda/code/bonde-client/node_modules/babylon/lib/index.js:1627:8)
    at Parser.pp$8.jsxParseIdentifier (/Users/gabrielrtakeda/code/bonde-client/node_modules/babylon/lib/index.js:6193:10)
    at Parser.pp$8.jsxParseNamespacedName (/Users/gabrielrtakeda/code/bonde-client/node_modules/babylon/lib/index.js:6204:19)
    at Parser.pp$8.jsxParseElementName (/Users/gabrielrtakeda/code/bonde-client/node_modules/babylon/lib/index.js:6219:19)
...
```